### PR TITLE
Utilities: Fixed buggy libFuzzer merge, which could skip new entries that increase coverage.

### PR DESCRIPTION
**Pull request by MikhailKrichanov**

### CODE REVIEW

File: User/Makefile [hunk 1]
```diff
@@ -101,8 +101,8 @@ ifeq ($(FUZZ_MEM),)
 endif
 
 ifeq ($(COVERAGE),1)
-	CFLAGS  += -fprofile-arcs -ftest-coverage -D COVERAGE_TEST
-	LDFLAGS += --coverage
```
Comment (vit9696) [inline User/Makefile:105]: This will not work on macOS, can we just not add `-Wl,-wrap,llvm_gcda_emit_arcs` for macOS?

Comment (vit9696) [inline User/Makefile:105]: This will not work on macOS, can we just not add `-Wl,-wrap,llvm_gcda_emit_arcs` for macOS?

File: User/Makefile [hunk 2]
```diff
@@ -101,8 +101,8 @@ ifeq ($(FUZZ_MEM),)
 endif
 
 ifeq ($(COVERAGE),1)
-	CFLAGS  += -fprofile-arcs -ftest-coverage -D COVERAGE_TEST
```
Comment (vit9696) [inline User/Makefile:104]: What is the difference between these two flags and the new one? Is it supported by gcc?

Comment (vit9696) [inline User/Makefile:104]: What is the difference between these two flags and the new one?

Comment (MikhailKrichanov) [reply User/Makefile:104]: --coverage is just a shorter synonym for those two options and yes, it is supported by gcc.
https://gcc.gnu.org/onlinedocs/gcc/Instrumentation-Options.html

**Pull request merged**